packages/openapi-to-graphql/docs/SUBSCRIPTIONS.md

```diff
@@ -0,0 +1,279 @@
+# Subscriptions with OpenAPI-to-GraphQL
```

Comment:
Should this tutorial live in the `openapi-to-graphql` package? Or in the monorepo like with the other tutorials/docs?

Reply:
If there's already a tutorial folder, it would make sense to put it here. I forgot about it since i only work on the package level.

Reply:
Okay I'll move it then!

Reply:
Actually I decide to keep it the same place.